transports/jms/src/test/resources/jms-redelivery-flow.xml

```diff
@@ -22,7 +22,7 @@
         <default-exception-strategy>
             <commit-transaction exception-pattern="org.mule.compatibility.transport.jms.redelivery.MessageRedeliveredException"/>
             <choice>
-                <when expression="#[payload.exception instanceof org.mule.compatibility.transport.jms.redelivery.MessageRedeliveredException]">
+                <when expression="#[payload.getException() instanceof org.mule.compatibility.transport.jms.redelivery.MessageRedeliveredException]">
```

Comment:
I think I had mistakenly changed it and to restore it back I used getException(), I'll check.